monai/networks/nets/ahnet.py

```diff
@@ -318,8 +318,8 @@ class AHNet(nn.Module):
     ``"transpose"`` rather than linear interpolations is faster. Therefore, this implementation sets ``"transpose"``
     as the default upsampling method.
 
-    To meet to requirements of the structure, for ``transpose`` mode, the input size of the first ``dim-1`` dimensions  should
-    be divisible by 2 ** (psp_block_num + 3) and no less than 32. For other modes, the input size of the first
+    To meet to requirements of the structure, for ``transpose`` mode, the input size of the first ``dim-1`` spatial dimensions
```

Comment:
May I know what the `dim-1` means here?

Thanks.

Reply:
Hi Nic, here if `spatial_dims=3`, `dim-1` means 2. I changed the description to avoid confusions.

Reply:
ok, looks good to me.
Thanks.